packages/xo-web/src/common/intl/messages.js

```diff
@@ -812,6 +812,7 @@ const messages = {
 
   // ----- Pool general -----
   earliestExpirationDate: 'Earliest expiration: {dateString}',
+  poolNoSupport: 'No support available for this pool',
```

Comment:
'No XCP-ng Pro support enabled on this pool" maybe?

Reply:
Even for non XCP-ng pools?